src/features/common/InputTypes/ToggleSwitch.tsx

```diff
@@ -1,22 +1,22 @@
-import { withStyles } from '@material-ui/core/styles';
-import Switch from '@material-ui/core/Switch';
+import { Switch } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import theme from '../../../theme/themeProperties';
 
 export default function ToggleSwitch(props: any) {
-  const ToggleSwitch = withStyles({
-    switchBase: {
-      color: '#fff',
-      '&$checked': {
-        color: props.color ? props.color : '#68B030',
-      },
-      '&$checked + $track': {
-        backgroundColor: props.color ? props.color : '#68B030',
-      },
+  const ToggleSwitch = styled(Switch)({
+    '& .MuiSwitch-switchBase.Mui-checked': {
+      color: props.color ? props.color : theme.primaryColor,
     },
```

Comment:
@Shreyaschorge This was added in with Sagar's merge commit. Probably needs to be removed as it breaks the code. Without these lines, component works correctly, and UI is fine as well.

Reply:
@Shreyaschorge I see that the following code isn't there in the conservationForms PR, so this can be ignored.

   ```
 checked: {},
    track: {
      backgroundColor: '#787878',
    },
  })(Switch);
```

Reply:
Okay